dataprofiler/profilers/data_labeler_column_profile.py

```diff
@@ -307,6 +307,48 @@ def profile(self) -> dict:
         }
         return profile
 
+    @classmethod
+    def load_from_dict(cls, data, options: dict | None = None) -> BaseColumnProfiler:
+        """
+        Parse attribute from json dictionary into self.
+
+        :param data: dictionary with attributes and values.
+        :type data: dict[string, Any]
+        :param options: options for loading column profiler params from dictionary
+        :type options: Dict | None
+
+        :return: Profiler with attributes populated.
+        :rtype: DataLabelerColumn
+        """
+        opt = DataLabelerOptions()
```

Comment:
if we want a labeler to be applicable across all cases. I assume it also comes through options here... I'm curious how we recognize it here in this `load_from_dict`. Does it come via
```python
{
    "DataLabelerColumn": {"data_labeler": ...},
    ...
}
```

What does the `...` look like?
is it similar where it says load_from_library? is it just the object?

Does it override the `data`?

Or does the data have precedence and if it exists in `options` it pulls from that. e.g. from_library matches the options `from_library` and the name is the same and has the object.
e.g.
```python
{
    "DataLabelerColumn": {"data_labeler": {"load_from_library": {"labeler_name": LabelerObject}}},
}
```

Reply:
@ksneab7 thoughts?